bescort.lic

```diff
@@ -301,6 +302,39 @@ class Bescort
 
   private
 
+  def use_flying_mount(type, mode, speed = 'fly')
+    case mode
+    when /^mount/
+      bput("get my #{@flying_mount}", /You get /, /You are already holding/)
```

Comment:
Since you're referencing `@flying_mount` here, why is it also passed in as the function argument `type`?

Reply:
It was from an earlier form of the method and I forgot to update it.  Done.